content/better-bibtex.ts

```diff
@@ -866,23 +866,15 @@ export class BetterBibTeX {
         // onRender: ({ body, item, editable, tabType }) => {
         onRender: ({ body, item, setSectionSummary }) => {
           const citekey = item.getField('citationKey')
-          const textbox = body.ownerDocument.getElementById('better-bibtex-citation-key')
+          const textbox = body.querySelector('#better-bibtex-citation-key')
           body.style.display = 'flex'
           // const was = textbox.dataset.itemid || '<node>'
           textbox.value = citekey || ''
           textbox.dataset.itemid = citekey ? `${item.id}` : ''
           setSectionSummary(citekey || '')
         },
-        onInit: ({ body, refresh }) => {
```

Comment:
The custom section will always be automatically re-rendered on item change. Not sure if I understand correctly, but I think this call to `refresh` is unnecessary unless you need it to re-render even when the item is not changed.

Reply:
The citation key can change when the formula changes and the key is refreshed, or when it is bumped out of the way by a freshly pinned key on another item, both of which happen without a change to the item whose key is changing.

Reply:
Ok, that makes sense. Sorry I'm not familiar with the BBT code.

Would you like me to update the PR for merging?

Reply:
Yeah I've already made the other change, thanks for that btw. I'm going to be bumping into more of these one-main-window assumptions, good to know what the fix is.

Reply:
> Ok, that makes sense. Sorry I'm not familiar with the BBT code.

Absolutely no worries, BBT has gotten quite complex over the years.